testserver/testserver.go

```diff
@@ -311,8 +315,13 @@ func (ts *testServerImpl) setPGURL(u *url.URL) {
 }
 
 // WaitForInit retries until a connection is successfully established.
-func (ts *testServerImpl) WaitForInit(db *sql.DB) error {
+func (ts *testServerImpl) WaitForInit() error {
```

Comment:
Slight nit that doesn't need to be fixed in this PR, but it makes more sense for this method to retain a handle to a `db` rather than one to a `testServerImpl` since it's not really used. Maybe this should be a package-level function that takes in a `db` rather than a method on `testServerImpl`.